pycc/ccresponse.py

```diff
@@ -1,13 +1,15 @@
 """
 ccresponse.py: CC Response Functions
 """
+from opt_einsum import contract
```

Comment:
This should be handled using something like `contract = self.ccwfn.contract` within the function that needs it.  This gives access to either the `opt_einsum` or PyTorch version.  See `solve_right()` for an example.

Reply:
Okay